bin/create.js

```diff
@@ -291,4 +352,104 @@ function parseHash (argv, field) {
         
         return hash;
     }, {});
-}
\ No newline at end of file
+}
+
+//
+// Load params and secrets from local config files (package.json & .env)
+//
+
+function parseEnv(key) {
+    var secrets = {};
+    var value   = ''
+
+    var noKeyErr = new Error('Non-existant secret specified, must be defined in .env or part of environment');
+
+    // Source from process.env
+    if(key)
+        Object
+            .keys(process.env)
+            .forEach(function (envKey) {
+                if(key === envKey) secrets[key] = process.env[key];
+            });
+
+    // Source from .env
+    var dotEnv;
+
+    try {
+        dotEnv = Fs.readFileSync('./.env', 'utf8');
```

Comment:
I think it would be great to use the `dotenv` module to do this for you, and then use `process.env`. This will also have the side effect of allowing people to use env variables in addition to `.env` file to provide parameters.